cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -555,8 +556,21 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
         Map<String, Object> optimizationProposal = new HashMap<>();
         optimizationProposal.put(CruiseControlRebalanceKeys.SUMMARY.getKey(),
                 proposalJson.getJsonObject(CruiseControlRebalanceKeys.SUMMARY.getKey()).getMap());
-        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
-
+//        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
+
+        String mapAsString = beforeAndAfterBrokerLoad.keySet().stream()
```

Comment:
Why are you converting the map into a string? Can't the `ConfigMapBuilder` just accept the map?

Reply:
The ConfigMapBuilder takes a Map but it should be of <String,String> type itself and our Map is like this 
 ```Map<Integer, Map<String, Object>>```